docs/configuration/configuration.md

```diff
@@ -757,6 +757,7 @@ Available meta labels:
 * `__meta_kubernetes_pod_container_port_name`: Name of the container port.
 * `__meta_kubernetes_pod_container_port_number`: Number of the container port.
 * `__meta_kubernetes_pod_container_port_protocol`: Protocol of the container port.
+* `__meta_kubernetes_pod_container_port_is_init`: Designates whether the container is an [InitContainer](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/)
```

Comment:
`__meta_kubernetes_pod_container_is_init` and please keep the list sorted.
I'd rephrase the description to "`true` if the container is an [InitContainer](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/)."